OutOfSchool/OutOfSchool.BusinessLogic/Services/SocialGroupService.cs

```diff
@@ -81,8 +94,7 @@ public async Task<SocialGroupDto> Create(SocialGroupCreate dto)
 
         logger.LogInformation($"SocialGroup with Id = {newSocialGroup?.Id} created successfully.");
 
-        return mapper.Map<SocialGroupDto>(socialGroup, opt =>
-        opt.Items["Localization"] = LocalizationType.Ua);
+        return mapper.Map<SocialGroupCreate>(newSocialGroup);
```

Comment:
You can use the custom code approach to create `SocialGroupDto` manually like you did before. Please don't return `...Create` types :)